docs/contributing-to-airbyte/README.md

```diff
@@ -4,88 +4,77 @@ description: "We love contributions to Airbyte, big or small."
+7. [Link an existing Issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) that does not include the `needs triage` label to your Pull Request. Pull requests without an issue attached to take longer to review.
 8. Write a PR title and description that follows the [Pull Request Handbook](./resources/pull-requests-handbook.md).
 9. An Airbyte maintainer will trigger the CI tests for you and review the code.
 10. Review and respond to feedback and questions by Airbyte maintainers.
 11. Merge the contribution.
 
-Pull Request reviews are done on a regular basis.
+Guidelines to common code contributions:
 
-:::info
-Please make sure you respond to our feedback/questions and sign our CLA.
+- [Contribute a code change to an existing connector](change-cdk-connector.md)
+- [Contribute a new connector](submit-new-connector.md)
 
-Pull Requests without updates will be closed due inactivity.
-:::
+## Documentation
 
-Guidelines to common code contributions:
+We welcome Pull Requests that enhance the grammar, structure, or fix typos in our documentation! Check the [Updating Documentation](writing-docs.md) guide for submitting documentation changes.
 
-- [Submit code change to existing Source Connector](change-cdk-connector.md)
-- [Submit a New Connector](submit-new-connector.md)
+If you're working on connector documentation, make sure you cover:
 
-## Documentation
+The following highlights from the [Google developer documentation style guide](https://developers.google.com/style) are helpful for new writers:
 
-We welcome Pull Requests that enhance the grammar, structure, or fix typos in our documentation.
+- [Be conversational and friendly without being frivolous](https://developers.google.com/style/tone)
+- [Use second person](https://developers.google.com/style/person)
+- [Use active voice](https://developers.google.com/style/voice)
+- [Put conditional clauses before instructions](https://developers.google.com/style/clause-order)
+- [Put UI elements in bold](https://developers.google.com/style/ui-elements)
+- [Write inclusive documentation](https://developers.google.com/style/inclusive-documentation)
+- [Don't pre-announce anything in documentation](https://developers.google.com/style/future)
 
-- Check the [Updating Documentation](writing-docs.md) guide for submitting documentation changes.
+Guideline for visuals: Use links to videos instead of screenshots (Reason: Users are more forgiving of outdated videos than screenshots).
 
-- Refer to this [template](https://hackmd.io/Bz75cgATSbm7DjrAqgl4rw) if you're submitting a new connector to the cataolg.
+For examples of good connector docs, see the [Salesforce source connector doc](https://docs.airbyte.com/integrations/sources/salesforce) and [Snowflake destination connector doc](https://docs.airbyte.com/integrations/destinations/snowflake)
```

Comment:
I was thinking that it would be useful to point out exemplary connector docs, so this is a nice addition!